src/conditional_layers/conditional_layer_hint.jl

```diff
@@ -96,10 +96,14 @@ function forward(X, Y, CH::ConditionalLayerHINT; logdet=nothing)
     # X-lane: conditional layer
     logdet ? (Zx, logdet3) = CH.CL_YX.forward(Yp, X)[2:3] : Zx = CH.CL_YX.forward(Yp, X)[2]
 
-    logdet ? (return Zx, Zy, logdet1 + logdet2 + logdet3) : (return Zx, Zy)
+    if x_lane == false
```

Comment:
```
logdet ? (return Zx, Zy, logdet1 + x_lane*ogdet2 + logdet3) : (return Zx, Zy)
```

Reply:
Yes, that is cleaner. Will change throughout